terraform/cyhy_nessus_ec2.tf

```
@@ -143,14 +143,18 @@ resource "aws_volume_attachment" "nessus_cyhy_runner_data_attachment" {
   instance_id = aws_instance.cyhy_nessus[count.index].id
 
   skip_destroy = true
-  depends_on   = [aws_ebs_volume.nessus_cyhy_runner_data]
 }
 
 # Provision a Nessus EC2 instance via Ansible
 module "cyhy_nessus_ansible_provisioner" {
   source = "github.com/cloudposse/terraform-null-ansible"
   count  = length(aws_instance.cyhy_nessus)
 
+  # Ensure all EBS volumes are attached before running Ansible
```

I'm fine either way here, but since there's only one:
```suggestion
  # Ensure the EBS volume is attached before running Ansible
```

---

I kind of like the boilerplate comments for this, but I see where you're coming from with your feedback. Thoughts on the following as something boilerplate-y but less quantity specific?

```suggestion
  # Ensure any EBS volumes are attached before running Ansible
```

---

Sure, that's a nice compromise.  👍

---

Please see [7a41648](https://github.com/cisagov/cyhy_amis/pull/422/commits/7a41648c06164c5973e68c7b7277f1974937a48e) and let me know if that works.